src/test/regress/expected/upgrade_columnar_metapage_after.out

```diff
@@ -80,7 +80,7 @@ SELECT columnar_storage_info('columnar_table_1');
 SELECT columnar_storage_info('columnar_table_2');
     columnar_storage_info
 ---------------------------------------------------------------------
- (2,0,10000000001,3,150902,26694)
+ (2,0,10000000001,3,150902,26443)
```

Comment:
I guess this will fail if/when we add citus 10.0 to the test matrix for upgrade tests.

We can possibly think about the problem when it happens because:
- we don't know if it we will add citus 10.0 to the test matrix
- the problem was already there

Reply:
I don't think this will differ in citus 10.

The change here is a side effect from the change in the before.

It used chunks of 100 with stripes of 1000. Since 10.1 we enforce the numbers to be in range 1000 to something high. To keep the change simple I have only bumped the chunk size to 1000 tuples.

Due to more tuples fitting in 1 compressed chunk the compression works better. The number increased here is the reserved byte offset in the storage manager for this relation. Given the same version of the compression libraries we can expect the same output sizes here. The used compression seems to be very deterministic, making it possible in the first place to embed these numbers in the test output.